decidim-meetings/config/locales/en.yml

```diff
@@ -360,9 +360,14 @@ en:
             location_hints_help: 'Location hints: additional info. Example: the floor of the building if it is an in-person meeting, or the meeting password if it is an online meeting with restricted access.'
             online_meeting_url_help: 'Link: allow participants to connect directly to your meeting'
             registration_url_help: 'Link: allow participants to go on the external service you are using for registrations'
+            reminder_enabled: Reminder enabled
+            reminder_help_text: Participants will receive an email the number of hours specified in advance of themeeting. Leave it empty or zero if no reminder is necessary.
```

Comment:
```suggestion
            reminder_help_text: Participants will receive an email the number of hours specified in advance of themeeting. If left empty or zero, the reminder will be sent at the default configured value of %{default_value} hours.
```

Reply:
@antopalidi the client has suggested that this should be the expected behavior. So let's change it to accomplish this in the rest of the code. Add a test case as well.